standings.js

```diff
@@ -61,13 +61,24 @@ fetchJSON("standings.json", function (gameResults) {
       return byWinningPercentage
     })
 
+const gamesBehindByTeam = sortedStandings.map((team) => {
+  const gamesBehind = ((sortedStandings[0].w - team.w) - (sortedStandings[0].l - team.l)) / 2
+  team.gb = gamesBehind
+  return gamesBehind
+}, {})
+
+  console.log(sortedStandings)
+  console.log(gamesBehindByTeam)
```

Comment:
These `console.log` statements should be removed from production code. It's generally considered bad practice, as it shows up in the user's console.

Reply:
Yup, I meant to do that, just forgot.